src/init.jl

```diff
@@ -1,13 +1,25 @@
-using Scratch
 using REPL
+import Base64
```

Comment:
```suggestion
import Base64
using Artifacts
```

Reply:
This should fix CI: https://github.com/JuliaPlots/Plots.jl/actions/runs/7759723372/job/21164417261?pr=4884#step:10:740